components/gcp/container/component_sdk/python/kfp_component/google/common/_utils.py

```diff
@@ -120,3 +121,54 @@ def wait_operation_done(get_operation, wait_interval):
             ))
         return operation
 
+
+def with_retries(
+    func: Callable,
+    on_error: Callable[[], Any] = None,
```

Comment:
nit: `Optional[...] = None,`

Reply:
Done. Thanks.
Interestingly, I remember python treating = None as Optional. in some cases. E.g. `var: Union[str, int] = None` -> `var: Union[str, int, None]` == `var: Optional[Union[str, int]]`